middleware/apps.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from django.apps import AppConfig
+logger = logging.getLogger(__name__)
+from django.core.cache import cache
+
+class MiddlewareConfig(AppConfig):
+    name = "middleware"
+    verbose_name = "Middleware"
+
+    def ready(self):
+        from middleware.tasks import retrieve_asset_config, store_camera_statuses
+
+        # countdown to get the middleware running for authentication
+
+        retrieve_asset_config.apply_async(countdown=2)
+        store_camera_statuses.apply_async()
```

Comment:
move this to schedules.py

Reply:
If these are required before startup, i would move them to the init script instead of here. A management command that is run before the server start makes more sense. 
@sainak if redis is down, this will cause the server to crash without the healthchecks running right ?

Reply:
@vigneshhari retrieve_asset_config requires the middleware server to be running for openid auth

@AnkurPrabhu we can remove these in favor of celery task, they both are fired as soon as celery is started